<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuryMainWindow</class>
 <widget class="QMainWindow" name="FuryMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="TestRender" name="openGLWidget">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Увеличение камеры</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="cameraZoomSlider">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>35</number>
          </property>
          <property name="maximum">
           <number>130</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carCamXLabel">
          <property name="text">
           <string>CarCam.x: 7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="carCamXSlider">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="carCamYLabel">
          <property name="text">
           <string>CarCam.y: 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="carCamYSlider">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="springLenghtLabel">
          <property name="text">
           <string>Длина пружин: 0.4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="springLenghtSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="springKLabel">
          <property name="text">
           <string>Жёсткость пружин: 500</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="springKSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shadowNearLabel">
          <property name="text">
           <string>ShadowNear: 0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="shadowNearSlider">
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shadowPlaneLabel">
          <property name="text">
           <string>ShadowPlane: 70</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="shadowPlaneSlider">
          <property name="value">
           <number>60</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shadowViewSizeLabel">
          <property name="text">
           <string>ShadowViewSize: 25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="shadowViewSizeSlider">
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="shadowCamDistanceLabel">
          <property name="text">
           <string>ShadowCamDistance: 37</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="shadowCamDistanceSlider">
          <property name="value">
           <number>64</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Нагрузка приложения на систему:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="nothingAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="quitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="nothingAction">
   <property name="text">
    <string>Ничего не делать</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>Об qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TestRender</class>
   <extends>QOpenGLWidget</extends>
   <header>TestRender.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
